RELEASE-NOTES.txt

```diff
@@ -2,7 +2,7 @@
 
 11.5
 -----
-- [*] Account deletion is now supported for all users in settings or in the empty stores screen. [https://github.com/woocommerce/woocommerce-ios/pull/8179]
+- [*] Account deletion is now supported for all users in settings or in the empty stores screen (in the ellipsis menu). [https://github.com/woocommerce/woocommerce-ios/pull/8179]
```

Comment:
Looks like we are supporting account creation for all users in the store picker and not just for ones with no stores. Should we update this message?
```suggestion
- [*] Account deletion is now supported for all users in settings or the store picker screen (in the ellipsis menu). [https://github.com/woocommerce/woocommerce-ios/pull/8179]
```

Reply:
Good catch! The original description (empty stores screen) is the expected behavior, since users with non-empty stores can continue with a store and then close their account in the settings. But I missed the logic to check on the empty stores state, and I just added it in https://github.com/woocommerce/woocommerce-ios/pull/8272/commits/5c0f97dfd4826725b58c55fd22a2043d123b85a8. I tested accounts with empty and non-empty stores.